tensorbay/exception.py

```diff
@@ -100,6 +102,114 @@ def __str__(self) -> str:
         return f"Invalid state code({self.status_code})! {self.response.url}"
 
 
+class AccessDeniedError(ResponseError):
```

Comment:
Do we need a distributor to raise different errors according to different status codes?